docs/about/index.rst

```diff
@@ -31,7 +31,7 @@ Migrating from Previous Versions
 --------------------------------
```

Comment:
i don't think we need this paragraph here now that we have a note in quickstart and user guide.